ext/http/http_next.rs

```diff
@@ -0,0 +1,800 @@
+use crate::LocalExecutor;
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
```

Comment:
This should throw of copyright checker - the copyright should be the first line in a file